.rubocop.yml

```diff
@@ -681,3 +685,149 @@ Style/IfUnlessModifierOfIfUnless:
 Style/ZeroLengthPredicate:
   Enabled: false
 
+# DISABLED - not useful
+Bundler/OrderedGems:
+  Enabled: false
+
+# do we require the encoding magic comments any more
+# DISABLED - TODO (2 entries)
+Layout/EmptyLineAfterMagicComment:
+  Enabled: false
+
+# extra new lines
+# DISABLED - TODO (2 entries)
+Layout/EmptyLinesAroundBeginBody:
+  Enabled: false
+
+# extra new lines
+# DISABLED - TODO (23 entries)
+Layout/EmptyLinesAroundExceptionHandlingKeywords:
+  Enabled: false
+
+# DISABLED - not useful
+Layout/IndentHeredoc:
+  Enabled: false
+
+# DISABLED - not useful
+Layout/MultilineArrayBraceLayout:
+  Enabled: false
+
+# DISABLED - not useful
+Layout/MultilineHashBraceLayout:
+  Enabled: false
+
+# DISABLED - not useful
+Layout/MultilineMethodCallBraceLayout:
+  Enabled: false
+
+# DISABLED - not useful
+Layout/MultilineMethodDefinitionBraceLayout:
+  Enabled: false
+
+# DISABLED - not useful
+Layout/SpaceInsidePercentLiteralDelimiters:
+  Enabled: false
+
+# DISABLED - TODO PUP-7948
+Lint/EmptyExpression:
+  Enabled: false
+
+# DISABLED - not useful
+Lint/EmptyWhen:
+  Enabled: false
+
+# DISABLED - TODO
+Lint/InheritException:
+  Enabled: false
+
+# DISABLED - TODO (1 entries)
+Lint/ScriptPermission:
+  Enabled: false
+
+# DISABLED - not useful
+Lint/ShadowedException:
+  Enabled: false
+
+# DISABLED - not useful
+Metrics/BlockLength:
+  Enabled: false
+
+# DISABLED - TODO PUP-8003
+Performance/Caller:
+  Enabled: false
+
+# DISABLED - TODO PUP-8003
+Performance/CompareWithBlock:
+  Enabled: false
+
+# DISABLED - TODO PUP-8004
+Security/JSONLoad:
+  Enabled: false
+
+# DISABLED - TODO PUP-8004
+Security/YAMLLoad:
+  Enabled: false
+
+# DISABLED - not useful
+Style/EmptyCaseCondition:
+  Enabled: false
+
+# DISABLED - not useful
+Style/EmptyMethod:
+  Enabled: false
+
+# DISABLED - not useful
+Style/FormatStringToken:
+  Enabled: false
+
+# DISABLED - TODO
+Style/InverseMethods:
+  Enabled: false
+
+# DISABLED - TODO (maybe)
+Style/MethodMissing:
+  Enabled: false
+
+# DISABLED - not useful
+Style/MixinGrouping:
+  Enabled: false
+
+# DISABLED - TODO (maybe)
+Style/MultilineIfModifier:
+  Enabled: false
+
+# DISABLED - not useful
+Style/MultilineMemoization:
+  Enabled: false
+
+# DISABLED - not useful
+Style/MultipleComparison:
+  Enabled: false
+
+# DISABLED - not useful
+Style/NumericLiteralPrefix:
+  Enabled: false
+
+# DISABLED - not useful
+Style/NumericPredicate:
+  Enabled: false
+
+# DISABLED - not useful
+Style/SymbolArray:
+  Enabled: false
+
+# DISABLED - not useful
+Style/TernaryParentheses:
+  Enabled: false
+
+# DISABLED - not useful
+Style/YodaCondition:
+  Enabled: false
+
+# DISABLED - code not ready for this
+GetText/DecorateString:
+  Enabled: false
+
+# DISABLED - code not ready for this
+GetText/DecorateFunctionMessage:
```

Comment:
These two `GetText/` are cops brought in by `rubocop-i18n`? Is enabling them part of [PUP-7954](https://tickets.puppetlabs.com/browse/PUP-7954)?

Reply:
I guess I could pull adding rubocop-i18n separate as we're going to have to disable everything and then make changes to the code base to add things back in.